pixeltable/utils/documents.py

```diff
@@ -18,85 +19,78 @@ class DocumentHandle:
     txt_doc: Optional[str] = None
 
 
-def get_document_handle(path: str) -> Optional[DocumentHandle]:
-    doc_format = puremagic.from_file(path)
+def get_document_handle(path: str) -> DocumentHandle:
+    _, extension = os.path.splitext(path)
+    handle = get_handle_by_extension(path, extension)
```

Comment:
Should this call be made if ext == ''?

Reply:
Yes I think it's fine, it will return `None` like with any unknown extension